client/gutenberg/extensions/map/settings.js

```diff
@@ -1,17 +1,24 @@
-/** @format */
-
+// Disable forbidden <svg> etc. so that frontend component does not depend on @wordpress/component
+/* eslint-disable react/forbid-elements */
 /**
  * External dependencies
  */
-
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 
 export const settings = {
 	name: 'map',
 	prefix: 'jetpack',
 	title: __( 'Map' ),
 	icon: (
-		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+		<svg
```

Comment:
Might be good to double "protect" this by adding a comment above it explaining why `SVG` primitives shall not be used.

Reply:
There's a comment at the top of the file with the `eslint-disable`. Do you think we need something here as well?

Reply:
I think we do — comments are read more likely if they're right next to code:

```
// disabled cuz...
/* disable rule */

...

// do not ever SVG here cuz...
<svg>
```

 — but feel free to ignore if you think it's better as it is now. :-)

Reply:
👍

Reply:
3924d0752bc7413312d0fc915983e48997bd0d54